app/models/course_module.rb

```diff
@@ -27,4 +29,16 @@ def lessons_with_progress(user)
       lesson
     end
   end
+
+  def user_progress(user)
+    lessons = lessons_with_progress(user)
+    if lessons.all? { |lesson| lesson.progress == "not_started" }
+      return "not_started"
+    end
+    if lessons.all? { |lesson| lesson.progress == "complete" }
+      return "complete"
+    end
+
+    "in_progress"
+  end
```

Comment:
Are we using this?

Reply:
doesn't look like we are. i'll remove it and give the app a run